README.md

```diff
@@ -2,35 +2,65 @@
 <img src="images/democracy-earth.png" width="400" title="Democracy Earth Foundation">
 </p>
 
-# Liquid Democracy
+# Liquid Democracy as a smart contract
 
 > 🚨🚧 _Everything in this repo is still a work in progress and higly experimental. It is not secure to use any of this code in production (mainnet) until proper security audits have been conducted._
 
-This is a smart contract implementation of a basic liquid democracy with three initial main functionalities: `vote()`, `delegate()` and `revoke()`. It mainly consists of `LiquidDemocracy.sol` and its corresponding test file `LiquidDemocracy.test.js`. This is the starting point of a roadmap that seeks to bridge Sovereign app with blockchain features. Please make sure you check out the [Waffle board](https://waffle.io/DemocracyEarth/contracts) and the [milestones](https://github.com/DemocracyEarth/contracts/milestones) for more information.
+This is a smart contract implementation (Dapp) in the Ethereum blockchain of a [liquid democracy](https://github.com/DemocracyEarth/paper) with three initial main functionalities: `vote()`, `delegate()` and `revoke()`. It mainly consists of `LiquidDemocracy.sol` and its corresponding test file `LiquidDemocracy.test.js`. This is the starting point of a roadmap that seeks to bridge [Sovereign app](https://github.com/DemocracyEarth/sovereign) with blockchain features. Please make sure you check out the [Waffle board](https://waffle.io/DemocracyEarth/contracts) and the [milestones](https://github.com/DemocracyEarth/dapp/milestones) for more information.
 
-## How to collaborate
+The most recent version of the dapp is available at http://dapp.democracy.earth. For desktop usage, you will need Metamask and a Metamask-supported browser to be able to interact with it. For mobile usage, use a dapp browser / wallet like [Toshi](https://www.toshi.org/).
```

Comment:
Reviewing this paragraph:

- Is the most recent version what's live at dapp.democracy.earth?
- Has anyone tested with Toshi for mobile?
- I edited the text in this paragraph a bit, feel free to edit further.

Reply:
@medied great points above. I especially think think we need to test this on Toshi, I don't think anybody has done this yet.

Reply:
I already tested it on Toshi, that's why I included it here. It works out of the box. We should try to aim    to have the latest version in dapp.democracy.earth even if it's not the case now.

Reply:
Good to hear you tested on Toshi 👍.

Who set up http://sovereign-dapp.herokuapp.com/ —was that @LucasIsasmendi?

Reply:
indeed